### DISCUSSION TITLE
Can OpenUtau have pitch bends and dynamics move with notes?

### DISCUSSION THREAD

**Discussion by sleepy7777** [stakira/OpenUtau#450] [Ideas]
I am new to get hub so if this is in the wrong spot then I am sorry. But I was wondering if maybe there would be a way to be able to move pitch bends and dynamic edits with the notes if you need to move them rather than drawing them out each time?

**Comment by nagotown** [comment]
i don't think any vocal synth has this feature and i assume it would be pretty hard to replicate

**Reply by sleepy7777** [reply]
would it be possible then to at least be able to copy and paste the pitch bends and dynamics to other parts of the ustx like how vocaloid let's you?

**Reply by nagotown** [reply]
i think stakira is either looking into it or putting it on the trello for now
